metaci/build/models.py

```diff
@@ -109,6 +110,8 @@ class Build(models.Model):
     log = models.TextField(null=True, blank=True)
     exception = models.TextField(null=True, blank=True)
     error_message = models.TextField(null=True, blank=True)
+    qa_comment = models.TextField(null=True, blank=True)
+    qa_user = models.ForeignKey('users.User', related_name='builds_qa', null=True, on_delete=models.PROTECT)
```

Comment:
related name isn't very fluent to read, qa_builds seems more logical...

Reply:
I personally like this format better though.  If you're looking through an alpha ordered list of attrs, you'd see builds and builds_qa next to each other which seems easier to work with.

Reply:
my only comment is musing on better django tooling to describe models the way desrcibe sobject works so that you see child relationships all together. BUT. that's not what we're here to discuss. fine by me. carry on.